index.d.ts

```diff
@@ -1,6 +1,15 @@
 import {type ChildProcess} from 'node:child_process';
 import {type Readable, type Writable} from 'node:stream';
 
+type NoOutputStdioOption =
+	| 'ignore'
+	| 'inherit'
+	| 'ipc'
+	| number
+	| Readable
+	| Writable
+	| [NoOutputStdioOption];
```

Comment:
`stdio` options which prevent Node.js from creating `childProcess.std*`, i.e. prevent us from retrieving the output.
Please note that when this does not apply when passing multiple values to this option. For example, `result.stdout` is `undefined` with `{ stdout: 'inherit' }` but not with `{ stdout: ['pipe', 'inherit'] }`. This PR accounts for this, and has a type test for it.